drracket/drracket/private/module-language-tools.rkt

```diff
@@ -662,9 +661,9 @@
       
       (define extra-default-filters '())
       (define default-extension "")
-      (define indentation-function (λ (x y) #f))
+      (define (indentation-function x y) #f)
       (define/public (get-indentation-function) indentation-function)
-      (define range-indentation-function (λ (x y z) #f))
+      (define (range-indentation-function x y z) #f)
```

Comment:
I think this refactoring should not be done (indeed it should go the other way) when the function ends up being a field declaration in a class. Thsi helps to make it clear to the programmer that they're using per-instance space, not per-class space.

Reply:
Hmm, I could get behind that. In this case it's also weird because of how the functions are named: `(define indentation-function (lambda (x y) #f)))` looks fine and `(define (indentation x y) #f)` looks fine, but `(define (indentation-function x y) #f)` looks weird.

Reply:
Great!

This also reminds me: there are definitely some bugs out there where people wrote `(define (f x ...) body)` inside a class an didn't realize that they are doing something bad and they should have written `(define/private (f x ...) body)`. If resyntax offers the "make a lambda to draw the eye"-type refactoring they might realize "oh! I wanted `define/private` instead". Or resyntax can even somehow tell them if it can comment on its suggestions somehow.

Reply:
Perhaps Resyntax could rewrite `(define (f x ...) body ...)` to `(define/private (f x ...) body ...)` whenever it's within a class and `f` is never mutated? Would that always be correct?